test/shortestpaths/dijkstra.jl

```diff
@@ -100,4 +100,171 @@
       dm = @inferred(dijkstra_shortest_paths(g,1;allpaths=true,trackvertices=true))
       @test dm.closest_vertices == [1,2,3,4,5]
     end
+
+
+
+    #Testing multisource On undirected Graph
+    g3 = PathGraph(5)
+    d = [ 0 1 2 3 4; 1 0 1 0 1; 2 1 0 11 12; 3 0 11 0 5; 4 1 19 5 0]
+    for g in testgraphs(g3)
+      z  = @inferred(floyd_warshall_shortest_paths(g, d))
+      zm = @inferred(multisource_dijkstra_shortest_paths(g, collect(1:5), d))
+      @test all(isapprox(z.dists,zm.dists))
+
+      for i in 1:5
+        state = dijkstra_shortest_paths(g,i;allpaths=true);
+        for j in 1:5
+          if z.parents[i,j]!=0
+            @test z.parents[i,j] in state.predecessors[j]
+          else
+            @test length(state.predecessors[j]) == 0
+            @test state.parents[j]==0
+          end
+        end
+      end
+
+      z  = @inferred(floyd_warshall_shortest_paths(g))
+      zm = @inferred(multisource_dijkstra_shortest_paths(g))
+      @test all(isapprox(z.dists,zm.dists))
+
+      for i in 1:5
+        state = dijkstra_shortest_paths(g,i;allpaths=true);
+        for j in 1:5
+          if z.parents[i,j]!=0
+            @test z.parents[i,j] in state.predecessors[j]
+          else
+            @test length(state.predecessors[j]) == 0
+          end
+        end
+      end
+
+      z  = @inferred(floyd_warshall_shortest_paths(g))
+      zm = @inferred(multisource_dijkstra_shortest_paths(g;parallel=true))
+      @test all(isapprox(z.dists,zm.dists))
+
+      for i in 1:5
+        state = dijkstra_shortest_paths(g,i;allpaths=true);
+        for j in 1:5
+          if z.parents[i,j]!=0
+            @test z.parents[i,j] in state.predecessors[j]
+          else
+            @test length(state.predecessors[j]) == 0
+          end
+        end
+      end
+
+      z  = @inferred(floyd_warshall_shortest_paths(g))
+      zm = @inferred(multisource_dijkstra_shortest_paths(g,[1,2]))
+      @test all(isapprox(z.dists[1:2,:],zm.dists))
+      for i in 1:2
+        state = dijkstra_shortest_paths(g,i;allpaths=true);
+        for j in 1:5
+          if z.parents[i,j]!=0
+            @test z.parents[i,j] in state.predecessors[j]
+          else
+            @test length(state.predecessors[j]) == 0
+          end
+        end
+      end
+    end
+
+
+    #Testing multisource On directed Graph
+    g3 = PathDiGraph(5)
+    d = float([ 0 1 2 3 4; 5 0 6 7 8; 9 10 0 11 12; 13 14 15 0 16; 17 18 19 20 0])
+
+    for g in testdigraphs(g3)
+      z  = @inferred(floyd_warshall_shortest_paths(g, d))
+      zm = @inferred(multisource_dijkstra_shortest_paths(g, collect(1:5), d))
+      @test all(isapprox(z.dists,zm.dists))
```

Comment:
General comment: please separate arguments to function calls with a comma followed by a space:
```
@test all(isapprox(z.dists, zm.dists))
```